### PR TITLE
feat(devtools): add searchbar

### PR DESCRIPTION
**Pull request by ferrettin**
#57 
Parrot Devtools

### CODE REVIEW

File: packages/parrot-devtools/__tests__/app/hooks/useScenarios.spec.js [hunk 1]
```diff
@@ -137,6 +139,15 @@ describe('useScenarios', () => {
     expect(global.fetch).toHaveBeenCalledTimes(2);
   });
 
+  it('should filter scenarios when filterValue is changed', () => {
```
Comment (infoxicator) [inline packages/parrot-devtools/__tests__/app/hooks/useScenarios.spec.js:142]: We probably need another happy path test that filters out and returns just the filtered values in the array rather than empty

File: packages/parrot-devtools/__tests__/app/components/ScenariosDisplay.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 American Express Travel Related Services Company, Inc.
```
[the remark this answers is not in the document]
Comment (nellyk) [reply packages/parrot-devtools/__tests__/app/components/ScenariosDisplay.spec.jsx:2]: yes, this need to be updated

**Comment by ferrettin** [comment]
Hey @nellyk and @infoxicator, thanks for taking a look at my PR. I've addressed the comments above, please let me know if any other changes need to be made
